auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -96,6 +99,19 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         }
 
         view.findViewById(R.id.button_next).setOnClickListener(this);
+
+        TextView termsText = view.<TextView>findViewById(R.id.email_tos_and_pp_text);
```

Comment:
optional style nit:

I normally see this:
```java
TextView termsText = (TextView) view.findViewById(R.id.email_tos_and_pp_text);
```

Not this:
```java
TextView termsText = view.<TextView>findViewById(R.id.email_tos_and_pp_text);
```

Reply:
Oh yeah, I forgot to mention something else about this. You don't need those casts! Notice how the IDE greys them out? That's cuz `findViewById` was updated to use generics in API 26.